source/Cosmos.Core_Plugs/System/DateTimeImpl.cs

```diff
@@ -1,68 +0,0 @@
-using System;
-using Cosmos.Core;
-using IL2CPU.API;
-using IL2CPU.API.Attribs;
-
-namespace Cosmos.Core_Plugs.System
-{
-    [Plug(Target = typeof(DateTime))]
-    public class DateTimeImpl
-    {
-
-        public static DateTime get_Now()
-        {
-            int[] raw = GetRawDate();
-
-            return new DateTime(
-                100 * BCDtoBIN(raw[10]) + BCDtoBIN(raw[9]), //YEAR
-                BCDtoBIN(raw[8]), //MONTH
-                BCDtoBIN(raw[7]), //DAY
-                BCDtoBIN(raw[4]), //HOUR
-                BCDtoBIN(raw[2]), //MINUTE
-                BCDtoBIN(raw[0]) //SECOND
-            );
-        }
-
-        // TODO: get timezone
-        public static DateTime get_UtcNow() => get_Now();
-
-        private static int[] GetRawDate()
-        {
-            IOPort p70 = new IOPort(0x70);
```

Comment:
I think these IOPorts are already in the RTC IOGroup, would it be better to just use those instead of making a new instance of these?

Reply:
Those are removed lines. I moved the plug to System and it uses RTC from HAL.

Reply:
Oh right didn't notice :') thanks for pointing that out